x-pack/plugins/fleet/server/services/output.test.ts

```diff
@@ -321,6 +325,42 @@ describe('Output Service', () => {
         { is_default: false }
       );
     });
+
+    // With logstash output
+    it('should throw if encryptedSavedObject is not configured', async () => {
+      const soClient = getMockedSoClient({});
+
+      await expect(
+        outputService.create(
+          soClient,
+          {
+            is_default: false,
+            is_default_monitoring: false,
+            name: 'Test',
+            type: 'logstash',
+          },
+          { id: 'output-test' }
+        )
+      ).rejects.toThrow(`Logstash output need encrypted saved object api key to be set`);
```

Comment:
```suggestion
      ).rejects.toThrow(`Logstash output needs encrypted saved object api key to be set`);
```

Reply:
Assuming this is for testing purposes and not user-facing, I won't edit it except to fix subject/verb agreement.